### PR TITLE
documentation for changing the datasource for consent management

### PR DESCRIPTION
**Pull request by somindatommy**
Documenting changing the datasource for consent management.

### CODE REVIEW

File: en/docs/setup/changing-datasource-consent-management.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+Identity and UMA data. However you can change this to any database type that is 
+supported by WSO2 Identity Server.
+
+!!! note
+    For more information about databases, see 
+    [Working with Databases](../../setup/working-with-databases) and 
+    [Setting Up Separate Databases for Clustering](../../setup/setting-up-separate-databases-for-clustering).
+
+Following are the sample configuration for each database type.
+
```
Comment (ayshsandu) [inline en/docs/setup/changing-datasource-consent-management.md]: Once when we separate out consent database from WSO2IdentityDB, we need to specify it in 
wso2is-5.9.0/repository/conf/consent-mgt-config.xml

--------
[authentication.consent]
data_source = "WSO2CONSENT_DB"
---
Add the toml config for that as well in each example

File: en/docs/setup/changing-datasource-consent-management.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    password = "regadmin"
+    driver = "org.postgresql.Driver"
+    jmx_enable=false
+    pool_options.maxActive = "80"
+    pool_options.maxWait = "60000"
+    pool_options.minIdle = "5"
+    pool_options.testOnBorrow = true
+    pool_options.validationQuery="SELECT 1"
+    pool_options.validationInterval="30000"
+    pool_options.defaultAutoCommit=false
```
Comment (ayshsandu) [inline en/docs/setup/changing-datasource-consent-management.md]: update changes related to [authentication.consent] data source here as well

File: en/docs/setup/changing-datasource-consent-management.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
+# Changing the Default Datasource for Consent Management
+
+WSO2 Identity Server is shipped with an embedded H2 database for storing data. 
+These default databases are located in the `<IS_HOME>/repository/database` directory 
+of the product pack.
+
+By default Consent Management data is stored in the identity database (IDENTITY_DB) with 
+Identity and UMA data. However you can change this to any database type that is 
+supported by WSO2 Identity Server.
+
```
Comment (ayshsandu) [inline en/docs/setup/changing-datasource-consent-management.md]: However, you can separate consent specific data into a separate data-source to any database type that is 
supported by WSO2 Identity Server.

Comment (somindatommy) [reply en/docs/setup/changing-datasource-consent-management.md]: Fixed : 380b00c31c88d7a4de4a940ca60310a3367e5c37

File: en/docs/setup/changing-datasource-consent-management.md (hunk 2 above)
Comment (somindatommy) [reply en/docs/setup/changing-datasource-consent-management.md]: Fixed : 380b00c31c88d7a4de4a940ca60310a3367e5c37

File: en/docs/setup/changing-datasource-consent-management.md (hunk 1 above)
Comment (somindatommy) [reply en/docs/setup/changing-datasource-consent-management.md]: Fixed : 380b00c31c88d7a4de4a940ca60310a3367e5c37

**Pull request merged**